src/foam/core/Validation.js

```diff
@@ -129,7 +129,7 @@ foam.CLASS({
             predicateFactory: function(e) {
               return e.GTE(foam.mlang.StringLength.create({ arg1: self }), self.minLength);
             },
-            errorString: `Please enter a ${this.label} with least ${this.minLength} character${this.minLength>1?'s':''}`
+            errorString: `Please enter a ${this.label.toLowerCase()} with least ${this.minLength} character${this.minLength>1?'s':''}`
```

Comment:
Same as above, here and below.

Reply:
also if sticking with the remainder then 
with least -> with at least